### PR TITLE
Task/rebrand

### PR DESCRIPTION
**Pull request by jackboberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,123 +124,123 @@ For Node.js projects, you can specify this within the engines block in the `pack
...
+status.xervo.io.
 
-    $ modulus status
+    $ xervo status
 
-##License
+## License
 
-Copyright (c) Modulus
+Copyright (c) Xervo
```
Comment (bsatrom) [inline README.md]: Copyright (c) Tangible

File: LICENSE-MIT [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Modulus
+Copyright (c) 2014 Xervo
```
Comment (bsatrom) [inline LICENSE-MIT]: (c) Tangible

**Comment by jackboberg** [comment]
@bsatrom I removed all the in-file blocks, and updated the two remaining `(c)` lines

File: lib/xervo.js [hunk 3]
```diff
@@ -13,13 +13,7 @@ xervo.program.Settings.autoHelp = false;
 xervo.io = require('./common/io');
 
 xervo.printHeader = function () {
-  xervo.io.print('   __    __   ______   _____    __  __   __       __  __   ______   '.verbose);
```
Comment (theworkflow) [inline lib/xervo.js]: ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                      ..               ;'''`                
                 ''''   `,,,,,                 ,,,,,   :''',    `''''',          ''''':  '''''''''''''''''   ''''''''''';       `'''''            `'''''       ,''''''.            ;'''`                
                 ''''    :::::`               .:::::   :''',     ''''''         .'''''   '''''''''''''''''   ''''''''''''''      '''''            ;'''''      ''''''''''           ;'''`                
                 ''''    ::::::               :::::`   :''',     ;'''''         ''''''   '''''''''''''''''   '''''''''''''''     '''''.           ''''',     ''''''''''''          ;'''`                
                 ''''     :::::               :::::    :''',      ''''',        ''''',   '''''''''''''''''   ''''''''''''''''    ''''';           '''''     ;'''''''''''';         ;'''`                
                 ''''     :::::`             ,:::::    :''',      ''''''       ,'''''    '''''''''''''''''   '''''''''''''''''   :'''''           '''''    `''''''''''''''         ;'''`                
                 ''''     ,:::::             :::::     :''',      ;'''''       ''''''    '''''''''''''''''   '''''''''''''''''   `'''''          `'''''    ''''''''''''''''        ;'''`                
                 ''''      :::::             :::::     :''',       ''''':      '''''.    '''''''''''''''''   ''''''''''''''''';   '''''          ,'''''   `''''''''''''''''`       ;'''`                
                 ''''      :::::.           :::::,     :''',       ''''''     :'''''     '''''''''''''''''   ''''''''''''''''''   '''''`         ''''':   ''''''''''''''''''       ;'''`                
                 ''''      ,:::::           :::::      :''',       :'''''     ''''''     '''''''''''''''''   ''''''''''''''''''   ''''':         '''''`   ''''''''..''''''''       ;'''`                
                 ''''       :::::     ,    `:::::      :''',        ''''':    '''''`     '''''               '''''`      ''''''`  ;'''''         '''''   :''''''.    ,'''''',      ;'''`                
                 ''''       :::::,    +    :::::,      :''',        ''''''   :'''''      '''''               '''''`       ''''':  `'''''         '''''   '''''':      ;''''''      ;'''`                
                 ''''       .:::::   ,+    :::::       :''',        :'''''   ''''''      '''''               '''''`       ''''''   '''''        .'''''   ''''''        ''''''      ;'''`                
                 ''''        :::::   ++:  .:::::       :''',         ''''';  '''''`      '''''               '''''`       .'''''   '''''        ;'''':  `''''',        ,'''''`     ;'''`                
                 ''''        :::::,  +++  :::::.       :''',         '''''' ;'''''       '''''               '''''`        '''''   ''''',       '''''`  ;'''''          ''''':     ;'''`                
                 ''''        `::::: :+++  :::::        :''',         ,''''' ''''';       '''''               '''''`        '''''   ;'''''       '''''   ''''''          ''''''     ;'''`                
                 ''''         ::::: ++++;.:::::        :''',          '''''''''''`       '''''               '''''`        '''''   .'''''       '''''   ''''',          ,'''''     ;'''`                
                 ''''         ::::::+++++:::::`        :''',          '''''''''''        '''''               '''''`        '''''    '''''      `'''''   '''''            '''''     ;'''`                
                 ''''          :::::+++++;::::         :''',          .''''''''';        '''''               '''''`        '''''    '''''      :'''';   '''''            '''''     ;'''`                
                 ''''          :::::'++++'::::         :''',           '''''''''         '''''               '''''`       :'''''    '''''`     '''''`  `'''''            '''''     ;'''`                
                 ''''          ,:::::+++++:::          :''',           '''''''''         ''''''''''''''''    '''''`       ''''';    ''''';     '''''   .'''''            '''''`    ;'''`                
                 ''''           :::::+++++;::          :''',           `''''''':         ''''''''''''''''    '''''`      .''''',    ,'''''     '''''   .'''''            '''''.    ;'''`                
                 ''''           :::::'+++++:,          :''',            '''''''          ''''''''''''''''    '''''`     ,''''''      '''''     '''''   ,'''''            ''''',    ;'''`                
                 ''''           ,:::::+++++:           :''',            '''''''          ''''''''''''''''    ''''''''''''''''''      '''''    ,'''';   :'''''            ''''',    ;'''`                
                 ''''            :::::+++++;           :''',            `''''';          ''''''''''''''''    ''''''''''''''''''      '''''`   '''''.   ;'''';            ''''':    ;'''`                
                 ''''            :::::'+++++           :''',            '''''''          ''''''''''''''''    '''''''''''''''''.      ''''',   '''''    :'''';            ''''':    ;'''`                
                 ''''           ,':::::+++++           :''',            '''''''          ''''''''''''''''    '''''''''''''''''       ,'''''   '''''    :'''''            ''''',    ;'''`                
                 ''''           ++:::::+++++:          :''',           `''''''''         ''''''''''''''''    ''''''''''''''''.       `'''''   '''''    ,'''''            ''''',    ;'''`                
                 ''''           ++;::::;+++++          :''',           '''''''''         ''''''''''''''''    ''''''''''''''';         '''''  `'''''    .'''''            '''''.    ;'''`                
                 ''''          :++':::::+++++          :''',           '''''''''`        '''''               '''''''''''''';          '''''  ;''''.    `'''''            '''''`    ;'''`                
                 ''''          ++++:::::'++++;         :''',          .''''''''''        '''''               '''''''''''''`           '''''. '''''     `'''''            '''''     ;'''`                
                 ''''          ++++;::::;+++++         :''',          '''''''''''        '''''               '''''`  ''''''           :''''; '''''      '''''            '''''     ;'''`                
                 ''''         ;++++':::::+++++`        :''',          ''''':'''''`       '''''               '''''`  :'''''           `''''' '''''      '''''            '''''     ;'''`                
                 ''''         +++++ :::::'++++'        :''',         ,''''' ''''''       '''''               '''''`   ''''',           '''''`'''''      ''''':          :'''''     ;'''`                
                 ''''        `+++++ ::::  +++++        :''',         '''''' ''''''       '''''               '''''`   ''''''           ''''','''',      ''''''          ''''''     ;'''`                
                 ''''        '++++:  :::  +++++`       :''',         '''''. `'''''.      '''''               '''''`   ,'''''           ''''''''''       ;'''''          ''''':     ;'''`                
                 ''''        +++++   ::,  '+++++       :''',        :'''''   ''''''      '''''               '''''`    ''''',          :'''''''''       `''''',        ,'''''      ;'''`                
                 ''''       .+++++   ,:    +++++       :''',        ''''''   ;'''''      '''''               '''''`    ''''''          `'''''''''        ''''''        ''''''      ;'''`                
                 ''''       '++++,    :    +++++,      :''',        '''''`   `''''',     '''''               '''''`    ,'''''           '''''''''        '''''':      ;''''''      ;'''`                
                 ''''       +++++     .    :+++++      :''',       ;'''''     ''''''     '''''               '''''`     ''''';          '''''''',        :'''''',    :'''''',      ;'''`                
                 ''''      ,+++++           +++++      :''',       ''''''     ;'''''     '''''''''''''''''   '''''`     ''''''          ''''''''`         '''''''',,''''''''       ;'''`                
                 ''''      +++++`           +++++:     :''',      `'''''`      ''''':    '''''''''''''''''   '''''`     .'''''          ;'''''''          ''''''''''''''''''       ;'''`                
                 ''''      +++++            ,+++++     :''',      ;'''''       ''''''    '''''''''''''''''   '''''`      ''''';         .'''''''          `''''''''''''''''`       ;'''`                
                 ''''     :++++'             +++++     :''',      ''''';       :'''''    '''''''''''''''''   '''''`      ''''''          '''''''           ''''''''''''''''        ;'''`                
                 ''''     +++++`             +++++;    :''',     `'''''`        ''''':   '''''''''''''''''   '''''`      .'''''          '''''':            ''''''''''''''         ;'''`                
                 ''''     +++++              .+++++    :''',     ''''''         ''''''   '''''''''''''''''   '''''`       ''''''         ''''''             ;'''''''''''':         ;'''`                
                 ''''    ;++++'               +++++    :''',     ''''';         ,'''''   '''''''''''''''''   '''''`       ''''''         ''''''              ''''''''''''          ;'''`                
                 ''''    +++++                '++++'   :''',    .'''''           ''''';  '''''''''''''''''   '''''`       `'''''`        .'''''               ''''''''''           ;'''`                
                 ''''   `+++++                `+++++   :''',    ''''''           ''''''  '''''''''''''''''   '''''`        ''''''         '''''                .''''''`            ;'''`                
                 ''''                                  :''',                                                                                                      ``               ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''                                  :''',                                                                                                                       ;'''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''`                
                                                                                                                                                                                                        
                                                                                                                                                                                                        
                                                                                                                                                                                                        
                                                                                                                                                                                                        
                                                                                                                                                                    ''                                  
                                                                                                                                                                    ''             .':                  
                                                                                                  ,,,,,`                 ,'.''    `',    '',                        ''             :'`                  
                                                                                                 .'''''`                 ;' ''    ,'     `;,                       `':             ''                   
                                                                                                 ;'''''                  ;; ''    ;'         ,',''  ;'`'  '',   ,'','.:'` ''  ''; ,'''.                 
                                                                                          ''''   '''''' ;'.  ;;;'   ',:; ;; ''''  ''  ''     ;'''', '':' ''''   '''''`''  '' ''''`;'''                  
                                                                                          '''',    ''  ;'''  ''''` ''''' '' ''''. '' ''''    '''''' ''''`''''' '''''' ''  '' '''''''''                  
                                                                                          '''''    ''  ,''':`'''':.''''' ''`''''' '' ''''    ''::'' ''''''. '' '' `'' '' .'::'; '  ''                   
                                                                                             ''    ''  `  ';.'';':'' `'' ':,': '' '':' `'    ''  '' ''. ''  '' ''  '' '' :'`''    `';                   
                                                                                          '''';   `':  .',':;' `',''  ''`'.;'  ''`','',,'    ''  '' ''  ''  '' ';  '' '' '' ''    ,'.                   
                                                                                         ''''',   .'. `''''.'' ,'`'' `'::' ''  ''.'`'''''   `';  ''.':  ''  ''`';  ';.'; '' ''    ;'                    
                                                                                         '' :'`   ;'  ;' ,' '' '' '' ''`'' ''  ';;' ''      .'' ,';;'`  '' `''`'' :',,'; '' ''    ''                    
                                                                                         '' ''    ''  '' '' '' '' ''''' '' '':''.'' '' :    ;'''''`''   '''''` ''''' ,''''' '''': '''`                  
                                                                                         ''`''    ''  ''''' '' '' .'''' '' ''''' '' :'''.   '''''' ''   :''''  ''''' `''''' ,'''' '''                   
                                                                                         '''''    ''  ,'''' ': ''  ` '' ''`','': ''  '''`   ''`''` ''    '''`  ;''''  ''.':  '''` :''                   
                                                                                         :'`''         ,          '`:''      `.       ,     ''

File: lib/controllers/.xervo-base-ignore [hunk 4]
```diff
@@ -1,8 +1,11 @@
 .git/
 .modulus-base-ignore
+.xervo-base-ignore
 .modulusignore
+.xervoignore
 .meteor/local/
 modulus-auto-zip-*
```
Comment (theworkflow) [inline lib/controllers/.xervo-base-ignore]: This file would be generated by the CLI [here](https://github.com/XervoIO/cli/blob/master/lib/controllers/project.js#L651). Wouldn't that make it ok to remove this line in this [commit](https://github.com/XervoIO/cli/pull/157/files#diff-e5001988c284bf49302c6986b4b88a3bL651)?

Comment (theworkflow) [inline lib/controllers/.xervo-base-ignore]: Same for the line `.modulusignore` and `modulus-auto-zip-*`

Comment (jackboberg) [inline lib/controllers/.xervo-base-ignore]: Those files may/probably exist on users machines now. In a future release we can deprecate them, and remove after that.

**Pull request merged**